packages/libs/extension-base/code-model-v1.d.ts

```diff
@@ -1,110 +1,91 @@
-declare type CollectionFormat =
-  'none' |
-  'csv' |
-  'ssv' |
-  'tsv' |
-  'pipes' |
-  'multi';
+declare type CollectionFormat = "none" | "csv" | "ssv" | "tsv" | "pipes" | "multi";
 
 declare type Constraint =
-  'None' |
-  'InclusiveMaximum' |
-  'ExclusiveMaximum' |
-  'InclusiveMinimum' |
-  'ExclusiveMinimum' |
-  'MaxLength' |
-  'MinLength' |
-  'Pattern' |
-  'MaxItems' |
-  'MinItems' |
-  'UniqueItems' |
-  'MultipleOf';
-
-declare type HttpMethod =
-  'get' |
-  'post' |
-  'put' |
-  'patch' |
-  'delete' |
-  'head' |
-  'options';
+  | "None"
+  | "InclusiveMaximum"
+  | "ExclusiveMaximum"
+  | "InclusiveMinimum"
+  | "ExclusiveMinimum"
+  | "MaxLength"
+  | "MinLength"
+  | "Pattern"
+  | "MaxItems"
+  | "MinItems"
+  | "UniqueItems"
+  | "MultipleOf";
+
+declare type HttpMethod = "get" | "post" | "put" | "patch" | "delete" | "head" | "options";
 
 declare type HttpStatusCode =
-  'Continue' |
```

Comment:
Ouch, the previous formatting of this makes me want to cry

Reply:
haha